ci-operator/step-registry/ipi/conf/vsphere/template/ipi-conf-vsphere-template-commands.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+
+echo "$(date -u --rfc-3339=seconds) - sourcing context from vsphere_context.sh..."
+# shellcheck source=/dev/null
+declare vsphere_datacenter
+declare vsphere_datastore
+declare vsphere_url
+declare vsphere_cluster
+source "${SHARED_DIR}/vsphere_context.sh"
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/govc.sh"
+if [ -z "${RHCOS_VM_TEMPLATE}" ]; then
+  installer_bin=$(which openshift-install)
+  ova_url=$("${installer_bin}" coreos print-stream-json | jq -r '.architectures.x86_64.artifacts.vmware.formats.ova.disk.location')
+  echo "${ova_url}" > "${SHARED_DIR}"/ova_url.txt
+  vm_template="${ova_url##*/}"
+
+# select a hardware version for testing
+  vsphere_version=$(govc about -json | jq -r .About.Version | awk -F'.' '{print $1}')
+  hw_versions=(15 17 18 19)
+  if [[ ${vsphere_version} -eq 8 ]]; then
+      hw_versions=(20)
+  fi
+  hw_available_versions=${#hw_versions[@]}
+  selected_hw_version_index=$((RANDOM % ${hw_available_versions}))
+  target_hw_version=${hw_versions[$selected_hw_version_index]}
+  echo "$(date -u --rfc-3339=seconds) - Selected hardware version ${target_hw_version}"
+  vm_template=${vm_template}-hw${target_hw_version}
+  
+  ENABLE_TEMPLATE="${SHARED_DIR}"/enable_template_content.txt
+  VERSION=$(echo "${JOB_NAME}" | grep -o -E '4\.[0-9]+')
+else
+  vm_template="${RHCOS_VM_TEMPLATE}"
+fi
+
+if [ ! -z ${VERSION} ]; then
+    Z_VERSION=$(echo ${VERSION} | cut -d'.' -f2)
+    if [ ${Z_VERSION} -gt 13  ]; then
+        cat >> "${ENABLE_TEMPLATE}" << EOF
```

Comment:
Suggest to directly use `"${SHARED_DIR}"/enable_template_content.txt` to replace `${ENABLE_TEMPLATE}`